### PR TITLE
Portfolio/popup window

### PR DESCRIPTION
**Pull request by Charzy-1**
I used Js DOM to create a pop-up window when my see project is clicked
I added another image to the section of the pop-up window
i made the window responsive to all possible given screens

### CODE REVIEW

**Comment by Charzy-1** [comment]
Modifications were made to correct linters error

**Review by basitali111 — CHANGES_REQUESTED** [review]
# Required Changes ♻️

Hi @Charzy-1 ,

- Please read the project requirements carefully before submitting your pull request. The requirements clearly mention that the popup should work for all projects in the portfolio section. Additionally, update your portfolio section to dynamically create all projects using a for loop or similar method.
**-In order to associate each project with the popup window details, you must refactor the project section:
o You need to use a JavaScript array to store all of the information for all projects.
o For each project you need to store the following pieces of data in a JavaScript object, at a minimum: name, description, featured image, technologies, link to live version, link to source.
o You need to update the main page so that the projects section is created dynamically using the information stored in that JavaScript object. Remember, all of the HTML in that section is created when the page loads.
o After that, you could implement the popup window.
• You should implement popups for both mobile and desktop screen sizes.**

Good job so far!
You still need to work on some issues to go to the next project, but you are almost there!
159483175-2f61461d-b352-4cb4-98f8-b173e05ab5ff
# Highlights
✔️ Professional Readme
✔️ Correct Gitflow
✔️ Linters pass
## Required 
- [ ] _Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me [@basitali111] in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

**Comment by Charzy-1** [comment]
Modifications were made in all three files to implement popup window.
open popup function was created with an argument of id passed into it to open popup in javascript
objects were creaated where captured dynamically using the id's

**Review by basitali111 — CHANGES_REQUESTED** [review]
# Required Changes ♻️

Hi @Charzy-1 ,
- Kindly remove the project card and popup from index.html and transfer them to your JS file. Kindly, tomorrow is just a demonstration, but please adhere strictly to the project requirements.
- "Also, when clicking on the 'see project' button, the page should move upward. Please fix this issue."
- Please read the project requirements carefully before submitting your pull request. The requirements clearly mention that the popup should work for all projects in the portfolio section. Additionally, update your portfolio section to dynamically create all projects using a for loop or similar method.
**-In order to associate each project with the popup window details, you must refactor the project section:
o You need to use a JavaScript array to store all of the information for all projects.
o For each project you need to store the following pieces of data in a JavaScript object, at a minimum: name, description, featured image, technologies, link to live version, link to source.
o You need to update the main page so that the projects section is created dynamically using the information stored in that JavaScript object. Remember, all of the HTML in that section is created when the page loads.
o After that, you could implement the popup window.
• You should implement popups for both mobile and desktop screen sizes.**

Good job so far!
You still need to work on some issues to go to the next project, but you are almost there!
159483175-2f61461d-b352-4cb4-98f8-b173e05ab5ff
# Highlights
✔️ Professional Readme
✔️ Correct Gitflow
✔️ Linters pass
## Required 
- [ ] _Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me [@basitali111] in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

**Comment by Charzy-1** [comment]
I have handled the popup which redirects the user to the begining of the page initially. It is fixed now

File: script.js [hunk 1]
```diff
@@ -9,4 +9,86 @@ hamburger.addEventListener('click', () => {
...
+    sourceLink: '#'
+  },
+  {
+    title: 'Blog',
+    description: 'This blog allows post to be shared',
+    imgSrc: './images/image5.jpg',
+    liveLink: '#',
+    sourceLink: '#'
+  }
+];
```
Comment (Pamphilemkp) [inline script.js]: - Please add a description paragraph on each card, when they get clicked they should appear the same as shown in the figma design, this is a simple task for you as you added it on the first item in the array as the only thing you can do  is to add the same key in the object called description and you can add some descriptions. with the same length as shown in the figma design.

**Review by Pamphilemkp — CHANGES_REQUESTED** [review]
## Hi :raising_hand: @Charzy-1 , 

Good job so far!
There are some issues that you still need to work on to prepare your project for the final evaluation but you are almost there!




## Highlights :green_circle: 


- PR is descriptive and professional :+1: 
- Popup windows are showing correctly and coming at the right position.
- No Linter error :heavy_check_mark: 


## Required Changes ♻️
- Please add a description paragraph on each card, when they get clicked they should appear the same as shown in the figma design, this is a simple task for you as the only thing you can do  is to add a new key in the object called description and you can add some description. Try to work on that.
Screenshot (286)
Screenshot (287)

- The section (My recent works) is overlapping when hitting the break point of about 1030px and above as shown below 
#Your design 
Screenshot (284)

#Original design
Screenshot (288)


_Check the comments under the review._

## Optional suggestions

- The cards after My recent works should only span one row for all the the screens with a fix size lesser than 768px.
Screenshot (289)



_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the (Code reviews limits policy) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using our Whatsapp Chill Academy Tutoring group

**Comment by Charzy-1** [comment]
I have added the description  to the cards. Please check

**Comment by Pamphilemkp** [comment]
> I have added the description to the cards. Please check
I hope you have implemented all the required changes i have raised

**Comment by Charzy-1** [comment]
I have effected the correction

**Review by Pamphilemkp — CHANGES_REQUESTED** [review]
## Hi :raising_hand: @Charzy-1 , 

Good job so far!
Your project is almost complete however
There are some issues that you still need to work on to prepare your project for the final evaluation but you are almost there!




## Highlights :green_circle: 


- PR is descriptive and professional :+1: 
- Popup windows are showing correctly and coming at the right position.  :heavy_check_mark: 
- Popup description paragraphs have been added successfully  :heavy_check_mark: 
- No Linter error :heavy_check_mark: 


## Required Changes ♻️

- The section (My recent works) is overlapping when hitting the break point of about 1030px and above as shown below 
## Your design 

Screenshot (284)

## Original design

Screenshot (288)

- All the popup cards are showing the same title and image while they should show the clicked title, description and image, only the first card under (My recent works) should show this popup below. Please kindy show the right image, title, description etc. when the user clicks any of the cards.

Screenshot (290)

 You are almost there, the card above should only be displayed for the first card the other remaining ones should the corresponding details.

_Check the comments under the review._

## Optional suggestions

- The cards after My recent works should only span one row for all the the screens with a fix size lesser than 768px.
Screenshot (289)



_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the (Code reviews limits policy) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using our Whatsapp Chill Academy Tutoring group

**Comment by Pamphilemkp** [comment]
Please before requesting for a new review make sure you implement the required changes. You are alomst there just repplace the popup cards details with the right ones and also fix the first card under My recent works

**Comment by Charzy-1** [comment]
Please check it now. I believe the issues raised has been handled.

**Review by Pamphilemkp — APPROVED** [review]
Hi @Charzy-1 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 




Congratulations! 🎉 
## Highlights
- No linter errors ✅  
- Good Pr title and description 💪🏿 
- The popup window is working 💪🏿 ✅  

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


- Please make sure the popup image block and paragraph have  the same height as shown in figma design. So the buttons (See Live & See Source) will end at the same level as the image and the paragraph will start also at the same level as the image block.

## Your design 

Screenshot (291)

## Figma design

Screenshot (292)

- The contact me section is overlapping when we hit the breakpoint of  769px to 853px, kindly make sure to fix this issue.
Screenshot (293)

- Kindly ensure that your application is deployed,  so users don't have to clone your repository to see your app, kindly checkout github pages.
[Deploy on Github Pages click here](https://www.youtube.com/watch?v=OltY8JIaP-4)

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification @Pamphilemkp 

------
_As described in the Code reviews limits policy  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, hit us up.

**Pull request merged**